azure-pipelines.release-vnext.yml

```diff
@@ -81,6 +81,11 @@ extends:
                   FLUENT_PROD_BUILD=true yarn nx run-many -t lint -p tag:vNext --exclude 'tag:tools,tag:type:stories,apps/**' --nxBail
                 displayName: lint
 
+              - script: |
+                  node -r ./scripts/ts-node/src/register ./scripts/executors/src/deprecate-react-components-preview-packages.ts --token $(npmToken)
+                displayName: 'Deprecate preview packages'
+                continueOnError: true
```

Comment:
it seems that with all the try-catches in place this will actually never fail even if it should. 

I suggest to either remove this and keep handling everything within the executor or not handle failures in the executor ( handle them but process.exit(1) when necessary ). I'm more inclined to 2nd approach, as a program should exit with 1 if there are issues, but pipeline flow should not. 

to add to this, I'm not really sure it's good idea to continue the release if deprecation failed as that would force us to do this manually - better approach is to re-run the release pipeline from my experience. 


thoughts?

Reply:
I'm hesitant to rerun the pipeline as there's a chance it might not fully succeed due to a potential package failure that could disrupt the release pipeline. However, it could be beneficial to try and then refine our strategy based on what we learn from the outcome.